### DISCUSSION TITLE
Enter a webpage with Ignore mode by default

### DISCUSSION THREAD

**Discussion by thepragmaticmero** [tridactyl/tridactyl#5035] [General]
Similar to Dark Reader. Always enter a certain webpage, ex. youtube.com, with Ignore mode by default.

What I mean by that is. Once I press the Ignore mode shortcut on a webpage, it should save the Ignore state on that website. I keep pressing "Shift + Escape" every time I open youtube and other webpages that have keyboard shortcuts by default (which are quicker than using tridactyl).

**Comment by bovine3dom** [comment]
Have you seen `:blacklistadd`?

**Reply by thepragmaticmero** [reply]
ye sorry haha, ok tnks
